src/index.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env browser */
+import { getWeather, getForecast, modal } from './api/weatherApi';
+
+const input = document.querySelector('input');
+const btn = document.querySelector('.searchButton');
+const close = document.querySelector('.closing');
+
+const search = () => {
+  getWeather(input.value);
+  getForecast(input.value);
```

Comment:
[Optional] It would be nice to make the two API calls simultaneously using `Promise.all`. This is totally optional.